src/scripts/training/lightgbm_python/train.py

```diff
@@ -68,6 +68,7 @@ def get_arg_parser(cls, parser=None):
         group_o = parser.add_argument_group(f"Outputs [{__name__}:{cls.__name__}]")
         group_o.add_argument("--export_model",
             required=False, type=str, help="Export the model in this location (file path)")
+        group_o.add_argument("--model_filename", required=False, type=str, help="Model filename")            
```

Comment:
set a default value here "model.txt" so you make sure you always have a value provided

Reply:
In this latest commit, I have default value in `spec.yaml` as well as argument in `train.py`. Since, we plan to use `spec.yaml` to generate documentation so I believe we wish to keep default value in both places?